forkit/tests/run.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python
+"""From http://stackoverflow.com/a/12260597/400691."""
+import argparse
+import sys
+
+import environ
+from django import setup
+from django.conf import settings
+from django.test.runner import DiscoverRunner
+from django.utils import six
+
+env = environ.Env()
+
+
+settings.configure(
+    CACHES={'default': {'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'}},
+    DATABASES={'default': env.db(default='postgres:///forkit')},
+    INSTALLED_APPS=('forkit', 'forkit.tests'),
+)
+
+
+setup()
+
+parser = argparse.ArgumentParser()
+parser.add_argument('tests', type=six.text_type, nargs='*', default=[])
+parser.add_argument('-v', '--verbosity', type=int, default=1)
+parser.add_argument('--failfast', action='store_true')
+parser.add_argument('--reverse', action='store_true')
```

Comment:
Why not `--keepdb`? This is a reason I prefer just creating a dummy project and using `manage.py test` - you don't need to faff about reimplementing functionality built in to  `manage.py test`.

Reply:
Although I must admit I can see the appeal... Perhaps I should give it a shot to see how it compares.

Reply:
This will require a larger change, and I'm keen to see this merged. I'm open to considering this change in future, but I'd like to leave it for the moment.

Reply:
I'm not well versed in django library test practices, but it seems to me that whacking pytest in here would be beneficial. Is that a done thing? It feels strange to be implementing arguments on a per-project basis.